pkg/workload/schemachange/query_util.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+package schemachange
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/cockroachdb/errors"
+	"github.com/jackc/pgx/v5"
+)
+
+const (
+	descriptorsJSON = `SELECT id, crdb_internal.pb_to_json('desc', descriptor) AS descriptor FROM system.descriptor`
```

Comment:
Nit: Consider renaming to descJSONQuery?

Reply:
Done!